packages/bot-skeleton/src/services/api/active-symbols.js

```diff
@@ -160,14 +186,97 @@ export default class ActiveSymbols {
                 if (submarket_name === submarket) {
                     const { symbols } = submarkets[submarket_name];
                     Object.keys(symbols).forEach(symbol_name => {
-                        accumulator.push([symbols[symbol_name].display_name, symbol_name]);
+                        const { display_name } = symbols[symbol_name];
+                        const symbol_display_name =
+                            display_name + (this.isSymbolClosed(symbol_name) ? ` ${localize('(Closed)')}` : '');
```

Comment:
```suggestion
               `${display_name}${this. isSymbolClosed(symbol_name)? localize('(Closed)') : ''}`;
```

Reply:
IMO the template literal makes it less clear. Won't change.